README.md

```diff
@@ -11,19 +11,52 @@ Runs tests. Tests are files ending in 'test.js'. Uses jest and runs in watch mod
 
 ### `yarn build`
 
-Builds the app for production to the `build` folder
+Builds the app for production to the `build` folder (which is ignored for the repo)
 
 ## Common Design Decisions
 
 - Avoid adding new dependencies. Minimal amount of dependencies is preferred. If a new dependency is needed, it must be approved and made a case of why it is needed.
-- All new features are added through a PR and need a code review with approval of dev lead.
+- All new features are added through a PR and need a code review with approval of the dev lead.
 - **NEVER** push directly to master.
+- Due to time constraints, tests are not required
```

Comment:
Even though no formal test are required, the developer must make sure the functionality is working properly before creating the pr.

Reply:
You are right. Added in b9d1045